collateralizedNFT.md

```diff
@@ -0,0 +1,500 @@
+---
```

Comment:
```suggestion
---
eip: 7595
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Sounds good!

Should we move the file to the ERC folder or just leave it in the current location (root directory)?

Reply:
Needs to be in the ERCs folder.